packages/frontend/amp/components/AdUserSync.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const preBidSrc =
+    'https://cdn.jsdelivr.net/npm/prebid-universal-creative@latest/dist/load-cookie.html';
+
+const preBidImg =
+    'data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==';
+
+export const AdUserSync: React.FC<{}> = () => {
+    return (
+        <>
```

Comment:
I don't think this is needed, as there's only one element returned?

Reply:
Good point. Originally there were supposed to be two but AMP won't let us have 2 tracking iframes. :(